gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -343,6 +343,15 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-alert-error">
               The color used on the <NxCode>NxAlert</NxCode> error icon and for other equivalent icons and purposes.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-binary-donut-chart-background-fill">
+              The color used on the <NxCode>NxBinaryDonutChart</NxCode> background.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-binary-donut-chart-background-stroke">
```

Comment:
This should always match the tile background color, correct? We should probably just use that variable then and not make this additional one.

Reply:
> This should always match the tile background color, correct? We should probably just use that variable then and not make this additional one.

Yup, changed here: 3c853a697ee170cbfbb5eb89bdff4b40691e67db